tests/test_models_dbi.py

```diff
@@ -46,6 +46,22 @@ def test_double_bracket_iteration_group_commutator(backend, nqubits):
     assert initial_off_diagonal_norm > dbf.off_diagonal_norm
 
 
+@pytest.mark.parametrize("nqubits", [3])
+def test_double_bracket_iteration_eval_dbr_unitary(backend, nqubits):
+    """ The bound is $$||e^{-[D,H]}-GC||\le s^{3/2}(||[H,[D,H]||+||[D,[D,H]]||$$"""
+    h0 = random_hermitian(2**nqubits, backend=backend)
+    d = backend.cast(np.diag(np.diag(backend.to_numpy(h0))))
+    dbi = DoubleBracketIteration(
+        Hamiltonian(nqubits, h0, backend=backend),
+        mode=DoubleBracketGeneratorType.group_commutator,
+    )
+
+    for s in np.linspace(0, 0.01, NSTEPS):
+        u = dbi.eval_dbr_unitary(s, mode=DoubleBracketRotationType.single_commutator)
+        v = dbi.eval_dbr_unitary(s, mode=DoubleBracketRotationType.group_commutator)
+
+        assert np.linalg.norm( u - v ) < 10 * s**(1.49) (  np.linalg.norm(h0) + np.linalg.norm(d)) * np.linalg.norm(h0) * np.linalg.norm(d)
```

Comment:
The bound in the DBI paper is $$||e^{-[D,H]}-GC||\le s^{3/2}(||[H,[D,H]||+||[D,[D,H]]||$$
and I'm taking an upper bound on that